packages/phone-number-privacy/combiner/src/pnp/endpoints/sign/io.ts

```diff
@@ -52,6 +52,11 @@ export class PnpSignIO extends IO<SignMessageRequest> {
     if (!super.inputChecks(request, response)) {
       return null
     }
+    // TODO: revisit extracting out duplicate init/validation code between sign/non-sign IO
```

Comment:
```suggestion
    // TODO(2.0.0): revisit extracting out duplicate init/validation code between sign/non-sign IO
```
Per my code cleanup PR that I'll share soon, let's try to use this TODO convention as much as possible